### ISSUE TITLE
qvm-backup doesn't check if there is enough space on dest if backing up to a VM

### ISSUE THREAD

**Issue by marmarek** [QubesOS/qubes-issues#864]
**Reported by joanna on 1 Jun 2014 09:51 UTC**
None

Migrated-From: https://wiki.qubes-os.org/ticket/864

**Comment by nothingmuch** [comment]
I've had two issues that I think are related, as based on the comment about compression, I believe this is mostly about clearly reporting errors, not so much predicting them

1. i tried backing up to FAT32 disk I had around, and at the 4G file limit the backup just failed, with no indication of what error occurred, 

2. Attempting a backup to ext4 failed immediately, again with no error reported, which `chown -R user:user` on the fresh ext4 partition rectified.

In fact, the second error has been plaguing me for a number of attempts over several weeks, and finally today when I accidentally tried a FAT32 disk and it partly worked, I only realized what was going on after i had partial success. I'm not sure why I assumed the backup would run as root in the target vm, but I do think there's room for UI improvements. I did not have time to debug the actual backup program and find out why errors were being ellided, unfortunately.
